test/models/tiktok_test.rb

```diff
@@ -41,6 +41,30 @@ class TiktokTest < ActiveSupport::TestCase
     assert_equal '@scout2015', data['username']
   end
 
+  test "should parse short TikTok link" do
+    m = create_media url: 'https://vt.tiktok.com/ZSduCHt6g/?k=1'
+    data = m.as_json
+    assert_equal 'item', data['type']
+    assert_match /Sabotage/, data['title']
+    assert_match /Michael/, data['author_name']
+    assert_equal '7090122043793984795', data['external_id']
+    assert_match 'https://www.tiktok.com/@ken28gallardo', data['author_url']
+    assert_nil data['error']
+    assert_equal '@ken28gallardo', data['username']
+  end
+end
+
+class TikTokUnitTest < ActiveSupport::TestCase
+  def setup
+    WebMock.enable!
```

Comment:
I'd like to move some of these request stubs up to the test_helper level eventually, but will take a bit of time - going to start trying to isolate tests as I touch the files.

Reply:
Looks good. Maybe eventually we can split those classes into different test files and then yes, that common logic could be in a `setup` method used by both.